fix(layout): ensure footer is positioned at the bottom of the page.

VictorArangoDev opened the pull request:
## 📄 Description  
Adjusted layout styles to guarantee the footer stays at the bottom of the viewport, regardless of the content height inside the main container.  

---

## 📝 Change summary  
- Updated layout styles to ensure footer alignment  
- Fixed positioning issue when content height was shorter than viewport  
- Improved consistency across different breakpoints  

---

## 📝 Types of changes  
- [x] 🐛 Bug fix (non-breaking change that fixes an issue)  
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to change)  
- [ ] ✨ New feature (non-breaking change that adds functionality)  
- [ ] ⏫ Version upgrade  
- [ ] 🧹 Refactor  
- [ ] 📝 Changes to documentation  

---

## 🧪 How to test  
1. Visit the following link: [http://localhost:4321/components/tooltip](http://localhost:4321/components/tooltip)  
2. Verify that the footer stays fixed at the bottom of the viewport regardless of the page content height  
3. Compare the layout with both short and long content to ensure consistency  

---

## 📸 Screenshots  


### Before  
<!-- Add screenshot here -->  

<img width="1514" height="1037" alt="Before" src="https://github.com/user-attachments/assets/8667ba20-403f-4d2a-8bb9-e18097a560e3" />

### After  
<!-- Add screenshot here -->  

<img width="1237" height="1018" alt="After" src="https://github.com/user-attachments/assets/91a2197a-6123-477f-9006-6d7052870173" />

---

VictorArangoDev committed:
fix(layout): ensure footer is positioned at the bottom of the page.
Adjusted layout styles to guarantee the footer stays at the bottom
of the viewport, regardless of the content height inside the main container.

Frontendland reviewed (changes requested):
Hey @victor-arango, thank you for your proposal! I left a couple of notes to ensure nothing breaks; otherwise, it's good to go.

VictorArangoDev committed:
fix(layout): ensure footer is positioned at the bottom of the page
Adjusted layout styles to guarantee the footer stays at the bottom
of the viewport, regardless of the content height inside the main container.

VictorArangoDev committed:
refactor(layout): move flex and width styles to .container selector

victor-arango commented:
Thanks for the feedback 🙌 I’ve moved the styles to the .container selector as suggested. This keeps .my focused only on margins. Appreciate the review!
